foundation/org.eclipse.persistence.core/src/main/java/org/eclipse/persistence/internal/dynamic/DynamicPropertiesInitializatonPolicy.java

```diff
@@ -78,6 +78,9 @@ else if (primClass == ClassConstants.PBYTE) {
                 value = Byte.MIN_VALUE;
             }
         }
+        else if (mapping.isCollectionMapping()) {
+           value = mapping.getContainerPolicy().containerInstance();
+        }
```

Comment:
this file needs an update to copyright year in the header (from 2021 to 2024)

Reply:
done